JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -312,6 +312,32 @@ module Microsoft.ApplicationInsights {
         }
 
         /**
+         * Sets the autheticated user id and the account id in this session.
+         * User auth id and account id should be of type string. They should not contain commas, semi-colons, equal signs, spaces, or vertical-bars.
+         *   
+         * @param authenticatedUserId {string} - The authenticated user id. A unique and persistent string that represents each authenticated user in the service.
+         * @param accountId {string} - An optional string to represent the account associated with the authenticated user.
+         */
+        public setAuthenticatedUserContext(authenticatedUserId: string, accountId?: string) {
```

Comment:
Agree with Max here - I think the setXXX/clearXXX api is clearer, even if less concise.